homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js

```diff
@@ -0,0 +1,32 @@
+function UserInfo() {
+    let fullName;
+    let phone;
+    Object.defineProperties(this, {
+        fullName: {
+            get() {
+                return fullName;
+            },
+            set(name) {
+                fullName = name.trim().toLowerCase().replace(/\s+/g, ' ').replace(/\b\w/g, letter => letter.toUpperCase());
```

Comment:
`replace(/\s+/g, ' ')` - replace whitespace with whitespace? cool :) but why is it needed here?

Reply:
I delete this regexp, but I wanted to remove more then one whitespace.